applications/GeoMechanicsApplication/custom_elements/fluid_body_flow_calculator.cpp

```diff
@@ -0,0 +1,61 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Richard Faasse
+//
+
+#include "fluid_body_flow_calculator.h"
+#include "custom_utilities/element_utilities.hpp"
+#include "includes/cfd_variables.h"
+
+namespace Kratos
+{
+
+FluidBodyFlowCalculator::FluidBodyFlowCalculator(InputProvider AnInputProvider)
+    : mInputProvider(std::move(AnInputProvider))
+{
+}
+
+std::optional<Matrix> FluidBodyFlowCalculator::LHSContribution() { return std::nullopt; }
+
+std::optional<Vector> FluidBodyFlowCalculator::RHSContribution()
+{
+    const auto    shape_function_gradients = mInputProvider.GetShapeFunctionGradients();
+    const Matrix& r_N_container            = mInputProvider.GetNContainer();
+
+    const auto integration_coefficients = mInputProvider.GetIntegrationCoefficients();
+
+    const auto& r_properties        = mInputProvider.GetElementProperties();
+    const auto  constitutive_matrix = GeoElementUtilities::FillPermeabilityMatrix(
+        r_properties, mInputProvider.GetLocalSpaceDimension());
+
+    RetentionLaw::Parameters RetentionParameters(r_properties);
```

Comment:
To follow the Kratos Style Guide:
```suggestion
   auto retention_law_parameters = RetentionLaw::Parameters{r_properties};
```

Reply:
Done